src/components/common/Button/index.tsx

```diff
@@ -35,16 +11,6 @@ export const ButtonBase = styled.button.attrs(({ $alt, $border, theme }: ButtonB
   transition: all 0.2s ease-in-out;
   transition-property: color, background-color, border-color, opacity;
 
-  &.cancel {
```

Comment:
the nice thing is these are all made irrelevant now with the theme + `kind={TYPE_OF_BUTTON}` prop use